python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/port-channel-interfaces.j2

```diff
@@ -373,31 +373,46 @@
 {%             set isis_instance = port_channel_interface.isis_enable | arista.avd.default("-") %}
 {%             set isis_bfd = port_channel_interface.isis_bfd | arista.avd.default("-") %}
 {%             set isis_metric = port_channel_interface.isis_metric | arista.avd.default("-") %}
 {%             set isis_circuit_type = port_channel_interface.isis_circuit_type | arista.avd.default("-") %}
 {%             set isis_hello_padding = port_channel_interface.isis_hello_padding | arista.avd.default("-") %}
-{%             set isis_authentication_mode = port_channel_interface.isis_authentication_mode | arista.avd.default("-") %}
+{%             if port_channel_interface.isis_authentication is arista.avd.defined %}
+{%                 if port_channel_interface.isis_authentication.both.mode is arista.avd.defined %}
+{%                     set isis_authentication_mode = port_channel_interface.isis_authentication.both.mode %}
+{%                 else %}
+{%                     if port_channel_interface.isis_authentication.level_1.mode is arista.avd.defined and port_channel_interface.isis_authentication.level_2.mode is arista.avd.defined %}
+{%                         set isis_authentication_mode = "Level-1: " ~ port_channel_interface.isis_authentication.level_1.mode ~ "<br>" ~ "Level-2: " ~ port_channel_interface.isis_authentication.level_2.mode %}
+{%                     elif port_channel_interface.isis_authentication.level_1.mode is arista.avd.defined %}
+{%                         set isis_authentication_mode = "Level-1: " ~ port_channel_interface.isis_authentication.level_1.mode %}
+{%                     elif port_channel_interface.isis_authentication.level_2.mode is arista.avd.defined %}
+{%                         set isis_authentication_mode = "Level-2: " ~ port_channel_interface.isis_authentication.level_2.mode %}
+{%                     endif %}
+{%                 endif %}
+{%             elif port_channel_interface.isis_authentication_mode is arista.avd.defined %}
+{%                 set isis_authentication_mode = port_channel_interface.isis_authentication_mode %}
+{%             endif %}
 {%             if port_channel_interface.isis_network_point_to_point is arista.avd.defined(true) %}
 {%                 set mode = "point-to-point" %}
 {%             elif port_channel_interface.isis_passive is arista.avd.defined(true) %}
 {%                 set mode = "passive" %}
 {%             else %}
 {%                 set mode = "-" %}
 {%             endif %}
-| {{ port_channel_interface.name }} | {{ isis_instance }} | {{ isis_bfd }} | {{ isis_metric }} | {{ mode }} | {{ isis_circuit_type }} | {{ isis_hello_padding }} | {{ isis_authentication_mode }} |
+| {{ port_channel_interface.name }} | {{ isis_instance }} | {{ isis_bfd }} | {{ isis_metric }} | {{ mode }} | {{ isis_circuit_type }} | {{ isis_hello_padding }} | {{ isis_authentication_mode | arista.avd.default("-") }} |
```

Comment:
Please set the default in above logic same as ethernet one or others

Reply:
Logic for ethernet and portchannel is different for this part. So it is simple to add default at the column level.

Reply:
Changed the code as per ethernet_interface doc template.